portfolio/src/main/webapp/index.html

```diff
@@ -7,31 +7,71 @@
     <script src="script.js"></script>
   </head>
   <body>
+
+    <!-- Sticky Navbar -->
     <div id="navbar">
         <div id="navbar-title">
             Mafe's Portfolio
         </div>
         <div id="navbar-right">
-            <a href="#aboutme">About Me</a>
-            <a href="#link2">Link 2</a>
-            <a href="#link3">Link 3</a>
+            <a href="#void">About Me</a>
```

Comment:
What's `#void`? Shouldn't these be `#aboutme` for the About Me section, `#projects` for Projects, etc.?

Reply:
I wanted to use JS to make the transitions in the links, but I haven't implemented that yet so I used #void as a dead link placeholder. In the future should I remove the link altogether or what is a good way to set placeholders for implementations that will come with the next commit?
I'll make sure to have this fixed with the next commit though!

Reply:
A plain `#` sign will do as a placeholder, though I would try to avoid adding placeholders, preferring to add functionality only when you need it. For example with these links, it would be better to keep them functioning by referencing the actual headers they refer to until the time that you get the smooth scrolling feature added.

Reply:
This makes perfect sense, thanks!